pages/login.vue

```diff
@@ -28,7 +28,7 @@
 
 <script>
 export default {
-  name: "details",
+  name: "login",
```

Comment:
I see some pages has "layout" instead of "name"?
I believe nuxt recommend "layout" so use that all time when working with nuxt